dev/test/document.ts

```diff
@@ -2004,3 +2008,51 @@ describe('listCollections() method', () => {
     });
   });
 });
+
+describe('withConverter() support', () => {
+  let firestore: Firestore;
+
+  beforeEach(() => {
+    return createInstance().then(firestoreInstance => {
+      firestore = firestoreInstance;
+    });
+  });
+
+  afterEach(() => verifyInstance(firestore));
+
+  it('for DocumentReference.withConverter().get()', async () => {
```

Comment:
s/DocumentReference.withConverter()/DocumentReference.get() since withConverter() is already part of the test name (via the describe).